packages/wrangler/src/__tests__/jest.setup.ts

```diff
@@ -105,3 +105,16 @@ jest.mock("../user/generate-random-state", () => {
 		generateRandomState: jest.fn().mockImplementation(() => "MOCK_STATE_PARAM"),
 	};
 });
+
+jest.mock("xdg-app-paths", () => {
+	return {
+		__esModule: true,
+		default: jest.fn().mockImplementation(() => {
+			return {
+				config() {
+					return "test-xdg-config";
```

Comment:
I feel that `config()` (even in a mock) should return an absolute path.

Reply:
Like a `/test-xdg-config` ?